### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [ahrm/sioyek#1005] [General]
Would be nice to get an official new release with maybe a change-log.
Last release was in 2022.
Thanks for a great PDF viewer!

**Comment by bkosowski** [comment]
@ahrm Hey!

What do you think about releasing a new desktop version? We know you wanted to pack of lot of stuff into the next release, but maybe a smaller incremental one would be fine?

Related topics:
#1156 
#734

**Reply by ahrm** [reply]
There is a preview release available here: https://github.com/ahrm/sioyek/releases/tag/sioyek3-alpha0.

**Reply by bkosowski** [reply]
Yes, indeed. But the version is one and a half years old. I'm wondering if a more up-to-date version could be compiled. Probably a lot has changed in the source code. But if during your development you can compile it for your testing purposes, maybe you would be inclined to upload it?
